mlos_bench/mlos_bench/util.py

```diff
@@ -69,13 +107,7 @@ def prepare_class_load(config: dict,
         Name of the class to instantiate and its configuration.
     """
     class_name = config["class"]
-    class_config = config.setdefault("config", {})
```

Comment:
nit: concise onliners are nice, but I don't think it's obvious when reading that `setdefault()` returns the key being set, so I kinda prefer this line that makes it more explicit that the `class_config` is its own entity and we're acting on merging that below rather than some ephemeral looking operation that's happening inside the merge_params(...) call.

Reply:
agree. Let me split it back into two lines here